app/src/main/java/com/duckduckgo/app/trackerdetection/TrackerDataLoader.kt

```diff
@@ -88,6 +90,7 @@ class TrackerDataLoader @Inject constructor(
             tdsEntityDao.updateAll(tdsJson.jsonToEntities())
             tdsDomainEntityDao.updateAll(tdsJson.jsonToDomainEntities())
             tdsTrackerDao.updateAll(tdsJson.jsonToTrackers().values)
+            tdsCnameEntityDao.updateAll(tdsJson.jsonToCnameEntities())
```

Comment:
Should we test that we are now doing this? :)

Reply:
I've added a test for this function (Class was untested).